Src/Microsoft.Dynamic/zlib.net/ZStream.cs

```diff
@@ -121,40 +121,22 @@ public sealed class ZStream
         /// Inflate object to perform data decompression
         /// </summary>
         private Inflate _istate;
-	
-        /// <summary>
-        /// Best guess about the data type: ascii or binary
-        /// </summary>
-        private BlockType data_type;
-		
-        /// <summary>
-        /// A checksum computed with Adler algorithm
-        /// </summary>
-        private long _adler;
 
-        #endregion
+	    #endregion
```

Comment:
Apparently there are some tabs from previous code. I will replace some of them with whitespace.